blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/server/BitbucketPipelineCreateRequest.java

```diff
@@ -0,0 +1,62 @@
+package io.jenkins.blueocean.blueocean_bitbucket_pipeline.server;
+
+import com.cloudbees.jenkins.plugins.bitbucket.BitbucketSCMSource;
+import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
+import com.google.common.collect.Lists;
+import io.jenkins.blueocean.commons.ErrorMessage;
+import io.jenkins.blueocean.credential.CredentialsUtils;
+import io.jenkins.blueocean.rest.impl.pipeline.credential.BlueOceanDomainRequirement;
+import io.jenkins.blueocean.rest.model.BlueScmConfig;
+import io.jenkins.blueocean.scm.api.AbstractMultiBranchCreateRequest;
+import jenkins.branch.MultiBranchProject;
+import jenkins.scm.api.SCMSource;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * @author Vivek Pandey
+ */
+public class BitbucketPipelineCreateRequest extends AbstractMultiBranchCreateRequest {
+    @DataBoundConstructor
+    public BitbucketPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
+        super(name, scmConfig);
+    }
+
+    @Override
+    protected SCMSource createSource(@Nonnull MultiBranchProject project, @Nonnull BlueScmConfig scmConfig) {
+        BitbucketSCMSource bitbucketSCMSource =  new BitbucketSCMSource(null, (String)scmConfig.getConfig().get("repoOwner"), (String)scmConfig.getConfig().get("repository"));
```

Comment:
Needs reworking for JENKINS-43507 see #1188 (which will give you a list of endpoints too)